vipps-login-api.md

```diff
@@ -1719,37 +1719,66 @@ Payload
 
 In addition to the normal Authentication, we offer [Partner keys](https://github.com/vippsas/vipps-partner/blob/main/partner-keys.md) which lets a partner make API calls on behalf of a merchant.
 
-Partner keys are currently only available for the Vipps Login from phone number flows:
+Partner keys are available for the following flows:
 
-* [Regular](#integrating-with-vipps-login-from-phone-number)
-* [Redirect to browser](#redirect-to-browser)
+* [Vipps Login in browser](#vipps-login-in-browser)
+* Vipps Login from phone number
+  * [Regular](#integrating-with-vipps-login-from-phone-number)
+  * [Redirect to browser](#redirect-to-browser)
+
+If you are a Vipps partner managing agreements on behalf of Vipps merchants, you can use your own API credentials to authenticate, and then send the Merchant Serial Number (MSN) to identify which of your merchants you are acting on behalf of.
 
-If you are a Vipps partner managing agreements on behalf of Vipps merchants, you can use your own API credentials to authenticate, and then send the `Merchant-Serial-Number` header to identify which of your merchants you are acting on behalf of. The `Merchant-Serial-Number` must be sent in the header of all API requests.
+Browser and phone number flows have minor differences and are described in more detail further down.
 
 Partners must fetch an access token to use as authentication.
 
-### Call guide
+### Partner key in the browser flow
+
+#### Authorization endpoint
+
+To initiate a login as partner, send Merchant Serial Number as a `msn` query parameter instead of `client_id`.
+
+```http request
+GET https://api.vipps.no/access-management-1.0/access/oauth2/auth?msn={Merchant-Serial-Number}&response_type=code&scope={scopes}&state={state}&redirect_uri={redirect_uri}
+```
+
+#### Token endpoint
 
 1. Fetch [access_token](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md#userinfo-call-by-call-guide)
 2. Use the access token in the `Authorization` header `Bearer access_token`
 3. Add the `Merchant-Serial-Number` header with the target merchant serial number
 
-### Example request
+**Example request:**
 
 ```http request
-POST https://api.vipps.no/vipps-login-ciba/api/backchannel/authentication
+POST https://api.vipps.no/access-management-1.0/access/oauth2/token
 Authorization: Bearer askdjhasdkjashjkdhaskjdh
 Content-Type: application/x-www-form-urlencoded
 Merchant-Serial-Number: 12345
 
-scope=name address openid&login_hint=urn:mobilenumber:{mobileNumber}&state=13821s837213bng26e2n61gege26&nonce=21hebdhwqdb7261bd1b23
+grant_type=authorization_code&code={authorization_code}&redirect_uri={redirect_uri}
 ```
 
-### Requirements
+### Partner in Vipps Login from phone number flow
 
-* The target client must be configured with `client_secret_basic` authentication.
+The `Merchant-Serial-Number` must be sent in the header of all API requests.
 
-### Where can it be used?
+#### Call guide
```

Comment:
Maybe "Call by call guide", like we have in other APIs?

Reply:
@cloveras 
Is it regarding the name or the actual call by call guide?

Reply:
I would leave like this, as it's a general call guide, not broken down into separate requests.